src/drive/web/modules/queries.js

```diff
@@ -325,6 +325,36 @@ export const buildEncryptionByIdQuery = id => ({
   }
 })
 
+/**
+ * Query to get all the files not trashed
+ * @returns {{definition: QueryDefinition}}
+ */
+export const buildSuggestionsQuery = () => ({
+  definition: Q(DOCTYPE_FILES)
+    .partialIndex({
+      _id: {
+        $ne: TRASH_DIR_ID
+      },
+      trashed: {
+        $or: [
+          {
+            $exists: false
+          },
+          {
+            $eq: false
+          }
+        ]
+      }
+    })
```

Comment:
I think we can also add something like: 

```
path: {
$or: [{
$exists: false
},{
$regex": "^(?!/.cozy_trash)
}
]
}
```

to remove all the folder having their path starting with .cozy_trash/

Reply:
Ok. I tried:
![Capture d’écran 2022-08-05 à 16 27 10](https://user-images.githubusercontent.com/8363334/183098300-ff85cd57-effc-4cbf-9e31-386a5b73bd66.png)

I think I will have a look with @paultranvan

Reply:
@Crash-- I'd assume we entirely rely on the `trashed` attribute, but we have issues with it, right? I'd prefer to fix those potential issues rather than complexify the index because of it

Reply:
We don't know how to fix this particular issue. We assume this is caused by the sharing protocole or by cozy-desktop. but we're not sure about it. This is tracked by our `data scrubber` and you can see that this is one of our top issues. 

<img width="975" alt="Capture d’écran 2022-08-08 à 10 20 03" src="https://user-images.githubusercontent.com/1107936/183372923-cc8bd81b-fb43-4bc6-a4b6-dcfa9d0777f6.png">

We fix issues when we find them (for instance https://github.com/cozy/cozy-stack/pull/3442) but we're not clean yet.

Reply:
changes made in https://github.com/cozy/cozy-drive/pull/2663/commits/59c86d5524666d92c6f711891919cb19813e0adc while @paultranvan is looking at the warning in cozy-client